bears/general/ImageDimensionBear.py

```diff
@@ -0,0 +1,53 @@
+import os
+import yaml
+
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from dependency_management.requirements.GemRequirement import GemRequirement
+from dependency_management.requirements.PipRequirement import PipRequirement
+from sarge import run, Capture
+
+
+class ImageDimensionBear(GlobalBear):
+    """
+    Checks the dimension of an image.
+
+    More information is available at <github.com/Abhi2424shek/img_checker>
+    """
+
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    REQUIREMENTS = {GemRequirement(' img_checker'),
+                    PipRequirement('pyyaml', '3.12')}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Image Dimension'}
+
+    def run(self,
+            image_config):
+
+        with open('img_config.yml', 'w') as yaml_file:
+            for each in image_config:
+                each = each.split(' ')
+                config = [{'directory': each[0],
+                           'width': int(each[1]),
+                           'height': int(each[2])}]
+                yaml.dump(config, yaml_file, default_flow_style=False)
+
+        cmd = 'ruby -r "img_checker.rb" -e '
+        cmd += '"ImgChecker.new.initialize img_config.yml"'
```

Comment:
I didn't up the version number, that's for them to decide. I just cleaned things up.